test/repository_test.rb

```diff
@@ -68,6 +73,12 @@
             UserRepository.persist(persisted_user)
             persisted_user.created_at.must_equal persisted_user.created_at
           end
+
+          it 'touches updated_at' do
+            user.name = 'My'
```

Comment:
If I don't change any attribute thus according to you does it make sense is "update at  persist method"?